spigot-impl/src/main/java/me/aurium/branch/spigot/message/AdventureMessage.java

```diff
@@ -0,0 +1,25 @@
+/*
+ *
+ *  Branch
+ *  Copyright © 2021 Aurium
+ *
+ *  Branch is free software: you can redistribute it and/or modify
+ *  It under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  Branch is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with Branch. If not, see <https://www.gnu.org/licenses/>
+ *  and navigate to version 3 of the GNU Affero General Public License.
+ *
+ */
+
+package me.aurium.branch.spigot.message;
```

Comment:
What purpose do AdventureMessage and FancyTextMessage serve?

Reply:
Unimplemented

Reply:
FancyTextMessage will allow for unified pretty formatting (e.g. all commands being red for name, grey for arguments, and a prefix of ""[KitPVP] /lah""

Reply:
Implementation will arrive soon!